app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".LoginActivity"></activity>
```

Comment:
Is this for the example tests? If so, add a comment for our future selfs so we know its not important